src/routes/error.ts

```diff
@@ -1,55 +1,85 @@
 // Copyright © 2022 Ory Corp
 // SPDX-License-Identifier: Apache-2.0
+import { FlowError, FrontendApi } from "@ory/client"
 import { UserErrorCard } from "@ory/elements-markup"
-import { AxiosError } from "axios"
 import {
-  defaultConfig,
-  isQuerySet,
   RouteCreator,
   RouteRegistrator,
+  defaultConfig,
+  isQuerySet,
 } from "../pkg"
 
+type OAuth2Error = {
+  error: string
+  error_description: string
+}
+
+function isOAuth2Error(query: qs.ParsedQs): query is OAuth2Error {
+  return query.error !== undefined && query.error_description !== undefined
```

Comment:
Can there be other parameters, as well? Is there a case where one of these wouldn't be present?

Reply:
unsure, might need to refer to hydra docs

Reply:
https://github.com/ory/fosite/blob/master/errors.go#L460

`error_hint` is super useful

Reply:
Nice, I removed the `error_description` requirement and added an option `error_hint`. We'll need to observe if anything else should be in there.